server/container/guice/mailet/src/main/java/org/apache/james/modules/server/MailetContainerModule.java

```diff
@@ -86,7 +86,7 @@ public class MailetContainerModule extends AbstractModule {
         "transport",
         All.class,
         RemoveMimeHeader.class,
-        pair -> Arrays.asList(pair.getMailet().getMailetConfig().getInitParameter("name").split(",")).contains("bcc"),
+        pair -> Arrays.asList(pair.getMailet().getMailetConfig().getInitParameter("name").toLowerCase().split(",")).contains("bcc"),
```

Comment:
Other stupid edge case question: do we handle the case where RemoveMimeHeader do not have a "name" property decently?

IE we likely throw today an uggly non explicit NPE, it would be nice to throw a contextualized with decent error message IllegalStateException, and have a unit test for it?

Reply:
Likely OK